test/mknod_test.c

```diff
@@ -28,8 +28,11 @@
 #include <string.h>
 #include <errno.h>
 #ifndef __APPLE__
+#elif
+#ifndef __FreeBSD__
```

Comment:
I am skeptical of code that CI does not exercise which will rot over time.  Is there some other way to approach this?

Reply:
I'm not sure what you mean, that file is Linux-kernel specific.

Reply:
Could we call `system` to invoke `mknod` instead?

Reply:
Thinking about this some more I think the `#ifdef` is fine.  But CI is broken so we cannot merge this.

Reply:
@diizzyy Could you move forward with this PR?

Reply:
I'll try to have a look again next week, unfortunately I don't have a Linux box available so I'm not sure why it fails.